### ISSUE TITLE
Abandon the old jboss server

### ISSUE THREAD

**Issue by wesleyhales** [wesleyhales/speedgun#19]
It works, but it's time to retire and rewrite the entire backend for speedgun.io in something else. Ideas?

**Comment by soulgalore** [comment]
haven't had any time to think of this. we'll be doing the same for sitespeed. to handle the queue we are planning to do it old classic way: store in database and pick up new jobs from there, but I guess there's better way to do it. would be nice to get rid of Java.

**Comment by wesleyhales** [comment]
@soulgalore yeah, I wanted to leverage JMS and the ease of using Resteasy (jaxrs) but I'm thinking that this is easier to do with a node middleware stack like strongloop or something else
